.claude/skills/worktree/SKILL.md

```
@@ -248,13 +248,44 @@ Directory suffix is auto-derived from the branch name:
+   wt.exe -w 0 new-tab --title "<slug>" -d "<forward-slash-path>"
+   ```
+
+   Use forward-slash paths (`C:/Users/Aurelio/synthorg-wt-<slug>`) to avoid Bash interpreting `\t` / `\U` / `\N` as escape sequences. `-w 0` targets the current Windows Terminal window. No trailing command -- see the "Design note (do NOT regress)" block under the `launch` subcommand below for why.
+
+   If not in Windows Terminal or `wt.exe` is missing, skip auto-launch and instead tell the user to run `cd <path> && claude` manually in each target terminal.
+
+7. **Present the output** to the user by printing each worktree's prompt INLINE in chat as a copy-pasteable fenced code block. Do NOT write the prompt to a file -- the user copies directly from chat.
+
+   Format per worktree:
+
+   ~~~
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)